CodeStarterCamp_Week2/Week2Step3.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  Week2Step3.swift
+//  CodeStarterCamp_Week2
+//
+//  Created by 홍세희 on 2023/08/29.
+//
+
+var allLottoNumberStorage: Dictionary<String , Set<Int>> = [String : Set<Int>]()
+var count: Int = 0
+
+func save(thisWeekLottoNumbers: Set<Int>) {
+    count += 1
+    allLottoNumberStorage["\(count)회차"] = thisWeekLottoNumbers
+}
+
+//알고 싶은 회차 입력 매개변수 -> "(알고싶은 회차의 숫자)회차"로 입력
+func findWhatLottoNumbers(week: String) {
+    if let _ = allLottoNumberStorage[week] {
+        var numberString: String
+        numberString = (allLottoNumberStorage[week]!).map({ String($0) }).joined(separator: ", ")
+        print(week + "의 로또 당첨 번호는 \(numberString) 입니다. ")
```

Comment:
이런이런~ 옵셔널 바인딩은 잘 해주셨는데 아래에서 강제추출을 사용하셨네요!
옵셔널 강제추출(!)은 사용을 지양하셔야 합니다. week에 잘못된 값이 들어온다면 앱이 종료되고 말거에요...
(지금은 if 에서 nil인 경우를 걸러주고 있어서 터지진 않습니다 😅)
옵셔널 바인딩으로 만들어진 상수를 통해 안전하게 추출된 값을 사용해주세요 ☺️

```suggestion
    if let lottoNumbers = allLottoNumberStorage[week] {
        let numberString = lottoNumbers.map({ String($0) }).joined(separator: ", ")
        print(week + "의 로또 당첨 번호는 \(numberString) 입니다. ")
```